apps/cowswap-frontend/src/modules/tradeWidgetAddons/containers/SettingsTab/index.tsx

```
@@ -99,6 +117,28 @@ export function SettingsTab({ className, recipientToggleState, hooksEnabledState
               </RowBetween>
 
+              {enablePartialApproval !== null && (
+                <RowBetween>
+                  <RowFixed>
+                    <ThemedText.Black fontWeight={400} fontSize={14}>
+                      <Trans>Enable partial approvals</Trans>
+                    </ThemedText.Black>
+                    <HelpTooltip
+                      text={
+                        <Trans>
+                          // todo fix text Allows you to set partial token approvals instead of full approvals.
```

Biome (2.1.2)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)